### PR TITLE
Replaced master with main

### PR DESCRIPTION
**Pull request by Saanidhyavats**
THIS REQUEST SOLVES #984 .
 In some code lines master was not replaced as it was indicating a path to a directory or link. Below is the output obtained by `git grep -n master`

.ci/dffml-install.sh:4:if [ "x${DFFML_RELEASE}" == "xmaster" ]; then
.ci/run.sh:161:  added_to_changelog=$(git diff origin/master --numstat -- CHANGELOG.md \
.ci/run.sh:216:  # Make master docs
.ci/run.sh:217:  master_docs="$(mktemp -d)"
.ci/run.sh:218:  TEMP_DIRS+=("${master_docs}")
.ci/run.sh:222:  mv pages "${master_docs}/html"
.ci/run.sh:249:  mv "${master_docs}/html" "${release_docs}/html/master"
.ci/run.sh:256:  mv build/ "${release_docs}/html/master/webui"
.ci/run.sh:266:  # Don't push docs unless we're running on master
.ci/run.sh:267:  if [ "x${GITHUB_ACTIONS}" == "xtrue" ] && [ "x${GITHUB_REF}" != "xrefs/heads/master" ]; then
.ci/run.sh:285:  git checkout master
.ci/run.sh:293:  docker build --build-arg DFFML_RELEASE=master -t intelotc/dffml .
CHANGELOG.md:272:  documentation for the master branch (on GitHub pages).
CONTRIBUTING.md:3:Please see the [contributing docs page](https://intel.github.io/dffml/master/contributing/).
Dockerfile:89:# The release of dffml we're using, latest is the latest PyPi release, master is
Dockerfile:90:# the master branch
README.md:3:[Actions Status](https://github.com/intel/dffml/actions)
README.md:5:[codecov](https://codecov.io/gh/intel/dffml)
README.md:37:Documentation for the master branch is hosted at
README.md:38:https://intel.github.io/dffml/master/index.html
README.md:42:The [contributing page](https://intel.github.io/dffml/master/contributing/index.html)
configloader/image/setup_common.py:42:    url=f"https://github.com/dffml/blob/master/configloader/image/README.md",
configloader/yaml/setup_common.py:42:    url="https://github.com/intel/dffml/blob/master/configloader/yaml/README.md",
dffml/df/memory.py:968:        # Acquire the master lock to find and or create needed locks
dffml/skel/model/README.md:9:Demo
docs/conf.py:5:# http://www.sphinx-doc.org/en/master/config
docs/conf.py:95:    "github_version": "master",
docs/contributing/dev_env.rst:35:   The version you want is https://intel.github.io/dffml/master/ if you are
docs/contributing/dev_env.rst:39:   looking at should say ``/master/`` in the URL. The URL you are currently on
docs/contributing/dev_env.rst:40:   should be https://intel.github.io/dffml/master/contributing/dev_env.html
docs/contributing/dev_env.rst:42:   This docs without ``/master/`` in the URL are built from the latest release.
docs/contributing/dev_env.rst:44:   The docs with ``/master/`` in the URL are built from the *master* branch.
docs/contributing/dev_env.rst:45:   The master branch is what you get when you clone the Git repo, which we're
docs/contributing/git.rst:26:    $ git checkout -b my_new_thing origin/master
docs/contributing/git.rst:99:latest changes from the master branch and ``merge`` or ``rebase`` them into your
docs/contributing/git.rst:143:|              | because you need to merge or rebase origin/master into your   |
docs/contributing/git.rst:149:- You need to grab the latests changes from the master branch. Maybe you need to
docs/contributing/gsoc/2019.md:59:- Follow the [README](https://github.com/intel/dffml/blob/master/README.md) and make sure you can run the tensorflow and git examples, Looking at the [Travis CI](https://
travis-ci.org/intel/dffml) may come in handy here.
docs/contributing/gsoc/2019.md:60:- [Run the tests](https://github.com/intel/dffml/tree/master/tests). DFFML has unit tests which are at about 90% coverage (amount of lines of code tested) for the main li
brary, the [Git features](https://github.com/intel/dffml/tree/master/feature/git/tests), and the [Tensorflow model](https://github.com/intel/dffml/tree/master/model/tensorflow/tests). Make sure you know h
ow to run them, and if you've never done Python unittests before you might want to [read up on python's unittest library](https://docs.python.org/3/library/unittest.html).  Figure out how to run a single 
test! Running one test instead of all of them will speed up your workflow when you are writing your tests!
docs/contributing/gsoc/2019.md:64:  - Write a [new feature](https://github.com/intel/dffml/issues/13)! Features can do anything you want, they generate some data based on a unique key, think of them like 
a scraper, see the [new feature guide](https://github.com/intel/dffml/tree/master/docs/tutorial/FEATURE.md) for more info. Make sure to include tests!
docs/contributing/gsoc/2019.md:65:  - Write a [new model](https://github.com/intel/dffml/issues/29)! Models are wrappers around any machine learning implementation or library, see the [new model guide](ht
tps://github.com/intel/dffml/tree/master/docs/tutorial/MODEL.md) for more info. Make sure to include tests!
docs/contributing/gsoc/2021.md:18:Read more on the about page: https://intel.github.io/dffml/master/about.html
docs/contributing/gsoc/2021.md:52:  [contributing page](https://intel.github.io/dffml/master/contributing/)
docs/contributing/gsoc/2021.md:63:- [Run the tests](https://github.com/intel/dffml/tree/master/tests). DFFML has
docs/contributing/gsoc/2021.md:91:    [new model tutorials](https://intel.github.io/dffml/master/tutorials/models/)
docs/contributing/gsoc/2021.md:115:Here's a [template](https://github.com/python-gsoc/python-gsoc.github.io/blob/master/2019/application2019.md)
docs/contributing/gsoc/2021.md:122:Please see the <a href="https://intel.github.io/dffml/master/contributing/gsoc/rubric.html">rubric</a> page for the project proposal
docs/contributing/maintainers.rst:55:- Switch to the master branch
docs/contributing/maintainers.rst:59:- Push the master branch
docs/contributing/maintainers.rst:73:    $ git checkout master
docs/examples/shouldi.rst:158:    `examples/shouldi <https://github.com/intel/dffml/blob/master/examples/shouldi/>`_
docs/examples/webhook/deploy.rst:13:    `examples/ffmpeg <https://github.com/intel/dffml/blob/master/examples/ffmpeg/>`_
docs/examples/webhook/deploy.rst:100:to be output to. An example video is available `here <https://github.com/intel/dffml/raw/master/examples/ffmpeg/input.mp4>`_ .
docs/index.rst:4:.. image:: https://github.com/intel/dffml/workflows/Tests/badge.svg?branch=master&event=push
docs/index.rst:7:.. image:: https://codecov.io/gh/intel/dffml/branch/master/graph/badge.svg
docs/index.rst:36:This is the documentation for the latest release, documentation for the master
docs/index.rst:37:branch can be found `here <master/index.html>`_.
docs/index.rst:82:    Master Branch Docs <https://intel.github.io/dffml/master/>
docs/installation.rst:116:the master branch.
docs/installation.rst:125:            $ python -m pip install -U "https://github.com/intel/dffml/archive/master.zip#egg=dffml"
docs/installation.rst:131:            (.venv) C:\Users\username> python -m pip install -U "https://github.com/intel/dffml/archive/master.zip#egg=dffml"
docs/installation.rst:144:            $ python -m pip install -U "https://github.com/intel/dffml/archive/master.zip#egg=dffml" \
docs/installation.rst:145:                "https://github.com/intel/dffml/archive/master.zip#egg=dffml-feature-git&subdirectory=feature/git"
docs/installation.rst:151:            (.venv) C:\Users\username> python -m pip install -U "https://github.com/intel/dffml/archive/master.zip#egg=dffml" ^
docs/installation.rst:152:                "https://github.com/intel/dffml/archive/master.zip#egg=dffml-feature-git&subdirectory=feature/git"
docs/installation.rst:156:right away. However, it comes with the master branch installed, you'll need to
docs/tutorials/dataflows/chatbot.rst:22:    `examples/dataflow/chatbot <https://github.com/intel/dffml/blob/master/examples/dataflow/chatbot>`_
docs/tutorials/dataflows/locking.rst:11:    `examples/dataflow/locking <https://github.com/intel/dffml/blob/master/examples/dataflow/locking>`_
docs/tutorials/models/docs.rst:180:`consoletest <https://github.com/intel/dffml/tree/master/dffml/util/testing/consoletest>`_
examples/shouldi/README.md:3:shouldi
examples/shouldi/README.md:53:  - [golangci-lint](https://github.com/golangci/golangci-lint/blob/master/README.md)
examples/shouldi/setup.cfg:11:url = https://github.com/intel/dffml/blob/master/examples/shouldi/README.md
examples/swportal/README.rst:8:`examples/swportal <https://github.com/intel/dffml/blob/master/examples/swportal/>`_
feature/auth/setup.py:29:    url="https://github.com/intel/dffml/blob/master/feature/auth/README.md",
feature/git/README.md:7:Demo
feature/git/setup.py:30:    url="https://github.com/intel/dffml/blob/master/feature/git/README.md",
model/autosklearn/dffml_model_autosklearn/__init__.py:25:    $ curl https://raw.githubusercontent.com/automl/auto-sklearn/master/requirements.txt | xargs -n 1 -L 1 python3 -m pip install
model/autosklearn/dffml_model_autosklearn/__init__.py:27:For more information about installation visit https://automl.github.io/auto-sklearn/master/installation.html#installation
model/autosklearn/dffml_model_autosklearn/autoregressor.py:50:    Implemented using `AutoSklearnRegressor <https://automl.github.io/auto-sklearn/master/api.html#regression>`_.
model/daal4py/README.md:9:Demo
model/scikit/setup.py:41:    url="https://github.com/intel/dffml/blob/master/model/scikit/README.md",
model/scratch/setup.py:32:    url="https://github.com/intel/dffml/blob/master/model/scratch/README.md",
model/spacy/setup_common.py:42:    url="https://github.com/intel/dffml/blob/master/model/spacy/README.md",
model/tensorflow/README.md:5:Demo
model/tensorflow/setup.py:30:    url="https://github.com/intel/dffml/blob/master/model/tensorflow/README.md",
model/tensorflow_hub/README.md:13:For tokenizatin needed by BERT models dffml_model_tensorflow_hub uses code from google research bert repository, License
model/tensorflow_hub/dffml_model_tensorflow_hub/tfhub_utils.py:3:https://github.com/google-research/bert/blob/master/tokenization.py.
model/tensorflow_hub/dffml_model_tensorflow_hub/tfhub_utils.py:5:https://github.com/google-research/bert/blob/master/LICENSE (Apache License 2.0)
model/tensorflow_hub/setup.py:30:    url="https://github.com/intel/dffml/blob/master/model/tensorflow_hub/README.md",
model/vowpalWabbit/setup.py:41:    url="https://github.com/intel/dffml/blob/master/model/vowpalWabbit/README.md",
news/origin_master_August_15_2019.txt:1:========================= origin/master news everyone! =========================
news/origin_master_August_15_2019.txt:5:happing lately in the master branch. We're due for a release soon! The next
news/origin_master_August_15_2019.txt:16:Labels for CSV sources were merged into master. Enabling users to save multiple
news/origin_master_August_15_2019.txt:50:learning library (sklearn) have been merged into master. With these additions
operations/binsec/setup.py:30:    url="https://github.com/intel/dffml/blob/master/operations/binsec/README.md",
operations/deploy/tests/test_operations.py:62:            "ref": "refs/master",
operations/deploy/tests/test_operations.py:65:                "default_branch": "master",
operations/image/dffml_operations_image/operations.py:56:    # Reference: https://docs.opencv.org/master/d8/d01/group__imgproc__color__conversions.html#ga4e0972be5de079fed4e3a10e24ef5ef0
scripts/check_literalincludes.py:47:        ["git", "diff-index", "origin/master"]
scripts/docs.sh:34:  curl -sSL -o pages/_static/copybutton.js "https://raw.githubusercontent.com/python/python-docs-theme/master/python_docs_theme/static/copybutton.js"
service/http/docs/conf.py:5:# http://www.sphinx-doc.org/en/master/config
service/http/setup.py:41:    url=f"https://github.com/{ORG}/dffml/tree/master/service/http",
source/mysql/setup_common.py:42:    url=f"https://github.com/{ORG}/dffml/blob/master/source/{NAME}/README.md",
tests/db/test_sqlite.py:43:                "SELECT count(name) FROM sqlite_master "
tests/operation/test_sqlite_query.py:85:                        "SELECT count(name) FROM sqlite_master "

### CODE REVIEW

**Comment by Saanidhyavats** [comment]
@pdxjohnny let me know if any change needs to be done

**Comment by pdxjohnny** [comment]
We're going to wait for #732 to be complete before merging this
